From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alex Henning <elcron@gmail.com>
Date: Fri, 17 Oct 2014 21:38:40 -0400
Subject: [PATCH 0413/6262] Removed vestigial POM file from NetworkTables

Flattened networktables/java/Athena into networktables/java since Athena
+ Desktop builds are identical and we dropped Azalea support.

Change-Id: If965891c163394fed3bd1a3570b1f923cdd6fd56
---
 networktables/java/Athena/pom.xml             | 26 ++++--
 networktables/java/pom.xml                    | 84 +++++++++++++++----
 .../stream/SocketServerStreamProvider.java    |  0
 .../networktables2/stream/SocketStream.java   |  0
 .../stream/SocketStreamFactory.java           |  0
 .../networktables2/stream/SocketStreams.java  |  0
 .../NetworkTableListenerTest.java             |  0
 .../networktables/NetworkTableTest.java       |  0
 .../networktables2/NetworkTableEntryTest.java |  0
 .../TransactionDirtierTest.java               |  0
 .../client/ClientConnectionAdapterTest.java   |  0
 .../ClientConnectionManagementTest.java       |  0
 .../ClientNetworkTableEntryStoreTest.java     |  0
 .../networktables2/client/ClientTest.java     |  0
 .../connection/BadMessageExceptionTest.java   |  0
 .../ConnectionMonitorThreadTest.java          |  0
 ...orkTableConnectionEntryAssignmentTest.java |  0
 ...NetworkTableConnectionEntryUpdateTest.java |  0
 .../NetworkTableConnectionTest.java           |  0
 .../server/ConnectionListTest.java            |  0
 .../server/ServerConnectionAdapterTest.java   |  0
 .../server/ServerIncomingMonitorTest.java     |  0
 .../networktables2/server/ServerTest.java     |  0
 .../server/ServerWriteManagerTest.java        |  0
 .../stream/SimpleIOStreamTest.java            |  0
 .../networktables2/stream/SocketTest.java     |  0
 .../networktables2/system/SystemTest.java     |  0
 .../thread/DefaultThreadManagerTest.java      |  0
 .../networktables2/type/ArrayTypeTest.java    |  0
 .../networktables2/type/ComplexTypeTest.java  |  0
 .../networktables2/type/EntryTypeTest.java    |  0
 .../wpilibj/networktables2/util/ListTest.java |  0
 .../networktables2/util/StackTest.java        |  0
 .../TableKeyNotDefinedExceptionTest.java      |  0
 .../test/java/test/util/ByteArrayStream.java  |  0
 .../java/test/util/LoggingOutputStream.java   |  0
 .../java/test/util/NetworkTableEntryUtil.java |  0
 .../java/test/util/NetworkTableTestNode.java  |  0
 .../java/test/util/StreamPipeProvider.java    |  0
 .../src/test/java/test/util/TestExecutor.java |  0
 .../util/TestServerConnectionProvider.java    |  0
 .../java/test/util/TestThreadManager.java     |  0
 42 files changed, 87 insertions(+), 23 deletions(-)
 rename networktables/java/{Athena => }/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketServerStreamProvider.java (100%)
 rename networktables/java/{Athena => }/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketStream.java (100%)
 rename networktables/java/{Athena => }/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketStreamFactory.java (100%)
 rename networktables/java/{Athena => }/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketStreams.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables/NetworkTableListenerTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables/NetworkTableTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/NetworkTableEntryTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/TransactionDirtierTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientConnectionAdapterTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientConnectionManagementTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientNetworkTableEntryStoreTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/BadMessageExceptionTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/ConnectionMonitorThreadTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/NetworkTableConnectionEntryAssignmentTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/NetworkTableConnectionEntryUpdateTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/NetworkTableConnectionTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ConnectionListTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerConnectionAdapterTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerIncomingMonitorTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerWriteManagerTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/stream/SimpleIOStreamTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/stream/SocketTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/system/SystemTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/thread/DefaultThreadManagerTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/type/ArrayTypeTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/type/ComplexTypeTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/type/EntryTypeTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/util/ListTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/networktables2/util/StackTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/edu/wpi/first/wpilibj/tables/TableKeyNotDefinedExceptionTest.java (100%)
 rename networktables/java/{Athena => }/src/test/java/test/util/ByteArrayStream.java (100%)
 rename networktables/java/{Athena => }/src/test/java/test/util/LoggingOutputStream.java (100%)
 rename networktables/java/{Athena => }/src/test/java/test/util/NetworkTableEntryUtil.java (100%)
 rename networktables/java/{Athena => }/src/test/java/test/util/NetworkTableTestNode.java (100%)
 rename networktables/java/{Athena => }/src/test/java/test/util/StreamPipeProvider.java (100%)
 rename networktables/java/{Athena => }/src/test/java/test/util/TestExecutor.java (100%)
 rename networktables/java/{Athena => }/src/test/java/test/util/TestServerConnectionProvider.java (100%)
 rename networktables/java/{Athena => }/src/test/java/test/util/TestThreadManager.java (100%)

diff --git a/networktables/java/Athena/pom.xml b/networktables/java/Athena/pom.xml
index edf532440d3058fabf59ba66b90dac38647046a6..7bce8f8eb3900e1964f1808502b10724382602fa 100644
--- a/networktables/java/Athena/pom.xml
+++ b/networktables/java/Athena/pom.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
-	 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
   <groupId>edu.wpi.first.wpilib.networktables.java</groupId>
   <artifactId>NetworkTables</artifactId>
@@ -38,11 +38,19 @@
     <profile>
       <id>docline-java8-disable</id>
       <activation>
-	<jdk>[1.8,</jdk>
+	    <jdk>[1.8,</jdk>
       </activation>
 
+      <distributionManagement>
+        <repository>
+          <id>filesystem.repo</id>
+          <name>Temporary Staging Repository</name>
+          <url>file://${user.home}/releases/maven</url>
+        </repository>
+      </distributionManagement>
+
       <build>
-	<plugins>
+	    <plugins>
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-javadoc-plugin</artifactId>
@@ -50,7 +58,7 @@
               <additionalparam>-Xdoclint:none</additionalparam>
             </configuration>
           </plugin>
-	</plugins>
+	    </plugins>
       </build>
     </profile>
   </profiles>
@@ -63,9 +71,9 @@
         <configuration>
           <source>1.6</source>
           <target>1.6</target>
-	  <testExcludes>
-	    <exclude>edu/wpi/first/wpilibj/networktables2/system/SystemTest.java</exclude>
-	  </testExcludes>
+	      <testExcludes>
+	        <exclude>edu/wpi/first/wpilibj/networktables2/system/SystemTest.java</exclude>
+	      </testExcludes>
         </configuration>
       </plugin>
       <plugin>
@@ -92,8 +100,8 @@
         <artifactId>maven-source-plugin</artifactId>
       </plugin>
       <plugin>
-	<groupId>org.apache.maven.plugins</groupId>
-	<artifactId>maven-javadoc-plugin</artifactId>
+	    <groupId>org.apache.maven.plugins</groupId>
+	    <artifactId>maven-javadoc-plugin</artifactId>
       </plugin>
     </plugins>
   </build>
diff --git a/networktables/java/pom.xml b/networktables/java/pom.xml
index 3258f857af755a1fc184cf103bf9ca359a4cc4c8..a3368d7d1eed12e8b62ee6230c60740aa6e770ae 100644
--- a/networktables/java/pom.xml
+++ b/networktables/java/pom.xml
@@ -1,25 +1,81 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
-	 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>edu.wpi.first.wpilib.networktables</groupId>
-  <artifactId>java</artifactId>
-  <packaging>pom</packaging>
-  <version>0.1.0-SNAPSHOT</version>
-  
-  <modules>
-    <module>Athena</module>
-  </modules>
+  <groupId>edu.wpi.first.wpilib.networktables.java</groupId>
+  <artifactId>NetworkTables</artifactId>
+  <packaging>jar</packaging>
+
+  <parent>
+    <groupId>edu.wpi.first.wpilib.templates.athena</groupId>
+    <artifactId>library-jar</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
+    <relativePath>../../maven-utilities/athena/library-jar</relativePath>
+  </parent>
 
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.11</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jmock</groupId>
+      <artifactId>jmock-junit4</artifactId>
+      <version>2.6.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jmock</groupId>
+      <artifactId>jmock-legacy</artifactId>
+      <version>2.6.0</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+  
   <profiles>
     <profile>
-      <id>desktop</id>
+      <id>docline-java8-disable</id>
       <activation>
-	<activeByDefault>true</activeByDefault>
+	    <jdk>[1.8,</jdk>
       </activation>
-      <modules>
-	<module>Athena</module>
-      </modules>
+
+      <build>
+	    <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-javadoc-plugin</artifactId>
+            <configuration>
+              <additionalparam>-Xdoclint:none</additionalparam>
+            </configuration>
+          </plugin>
+	    </plugins>
+      </build>
     </profile>
   </profiles>
+  
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.1</version>
+        <configuration>
+          <source>1.6</source>
+          <target>1.6</target>
+	      <testExcludes>
+	        <exclude>edu/wpi/first/wpilibj/networktables2/system/SystemTest.java</exclude>
+	      </testExcludes>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-source-plugin</artifactId>
+      </plugin>
+      <plugin>
+	    <groupId>org.apache.maven.plugins</groupId>
+	    <artifactId>maven-javadoc-plugin</artifactId>
+      </plugin>
+    </plugins>
+  </build>
 </project>
diff --git a/networktables/java/Athena/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketServerStreamProvider.java b/networktables/java/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketServerStreamProvider.java
similarity index 100%
rename from networktables/java/Athena/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketServerStreamProvider.java
rename to networktables/java/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketServerStreamProvider.java
diff --git a/networktables/java/Athena/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketStream.java b/networktables/java/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketStream.java
similarity index 100%
rename from networktables/java/Athena/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketStream.java
rename to networktables/java/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketStream.java
diff --git a/networktables/java/Athena/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketStreamFactory.java b/networktables/java/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketStreamFactory.java
similarity index 100%
rename from networktables/java/Athena/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketStreamFactory.java
rename to networktables/java/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketStreamFactory.java
diff --git a/networktables/java/Athena/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketStreams.java b/networktables/java/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketStreams.java
similarity index 100%
rename from networktables/java/Athena/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketStreams.java
rename to networktables/java/src/main/java/edu/wpi/first/wpilibj/networktables2/stream/SocketStreams.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables/NetworkTableListenerTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables/NetworkTableListenerTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables/NetworkTableListenerTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables/NetworkTableListenerTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables/NetworkTableTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables/NetworkTableTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables/NetworkTableTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables/NetworkTableTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/NetworkTableEntryTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/NetworkTableEntryTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/NetworkTableEntryTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/NetworkTableEntryTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/TransactionDirtierTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/TransactionDirtierTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/TransactionDirtierTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/TransactionDirtierTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientConnectionAdapterTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientConnectionAdapterTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientConnectionAdapterTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientConnectionAdapterTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientConnectionManagementTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientConnectionManagementTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientConnectionManagementTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientConnectionManagementTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientNetworkTableEntryStoreTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientNetworkTableEntryStoreTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientNetworkTableEntryStoreTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientNetworkTableEntryStoreTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/client/ClientTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/BadMessageExceptionTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/BadMessageExceptionTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/BadMessageExceptionTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/BadMessageExceptionTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/ConnectionMonitorThreadTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/ConnectionMonitorThreadTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/ConnectionMonitorThreadTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/ConnectionMonitorThreadTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/NetworkTableConnectionEntryAssignmentTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/NetworkTableConnectionEntryAssignmentTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/NetworkTableConnectionEntryAssignmentTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/NetworkTableConnectionEntryAssignmentTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/NetworkTableConnectionEntryUpdateTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/NetworkTableConnectionEntryUpdateTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/NetworkTableConnectionEntryUpdateTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/NetworkTableConnectionEntryUpdateTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/NetworkTableConnectionTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/NetworkTableConnectionTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/NetworkTableConnectionTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/connection/NetworkTableConnectionTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ConnectionListTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ConnectionListTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ConnectionListTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ConnectionListTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerConnectionAdapterTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerConnectionAdapterTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerConnectionAdapterTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerConnectionAdapterTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerIncomingMonitorTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerIncomingMonitorTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerIncomingMonitorTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerIncomingMonitorTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerWriteManagerTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerWriteManagerTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerWriteManagerTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/server/ServerWriteManagerTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/stream/SimpleIOStreamTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/stream/SimpleIOStreamTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/stream/SimpleIOStreamTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/stream/SimpleIOStreamTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/stream/SocketTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/stream/SocketTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/stream/SocketTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/stream/SocketTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/system/SystemTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/system/SystemTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/system/SystemTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/system/SystemTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/thread/DefaultThreadManagerTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/thread/DefaultThreadManagerTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/thread/DefaultThreadManagerTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/thread/DefaultThreadManagerTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/type/ArrayTypeTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/type/ArrayTypeTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/type/ArrayTypeTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/type/ArrayTypeTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/type/ComplexTypeTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/type/ComplexTypeTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/type/ComplexTypeTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/type/ComplexTypeTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/type/EntryTypeTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/type/EntryTypeTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/type/EntryTypeTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/type/EntryTypeTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/util/ListTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/util/ListTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/util/ListTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/util/ListTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/util/StackTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/util/StackTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/networktables2/util/StackTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/networktables2/util/StackTest.java
diff --git a/networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/tables/TableKeyNotDefinedExceptionTest.java b/networktables/java/src/test/java/edu/wpi/first/wpilibj/tables/TableKeyNotDefinedExceptionTest.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/edu/wpi/first/wpilibj/tables/TableKeyNotDefinedExceptionTest.java
rename to networktables/java/src/test/java/edu/wpi/first/wpilibj/tables/TableKeyNotDefinedExceptionTest.java
diff --git a/networktables/java/Athena/src/test/java/test/util/ByteArrayStream.java b/networktables/java/src/test/java/test/util/ByteArrayStream.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/test/util/ByteArrayStream.java
rename to networktables/java/src/test/java/test/util/ByteArrayStream.java
diff --git a/networktables/java/Athena/src/test/java/test/util/LoggingOutputStream.java b/networktables/java/src/test/java/test/util/LoggingOutputStream.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/test/util/LoggingOutputStream.java
rename to networktables/java/src/test/java/test/util/LoggingOutputStream.java
diff --git a/networktables/java/Athena/src/test/java/test/util/NetworkTableEntryUtil.java b/networktables/java/src/test/java/test/util/NetworkTableEntryUtil.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/test/util/NetworkTableEntryUtil.java
rename to networktables/java/src/test/java/test/util/NetworkTableEntryUtil.java
diff --git a/networktables/java/Athena/src/test/java/test/util/NetworkTableTestNode.java b/networktables/java/src/test/java/test/util/NetworkTableTestNode.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/test/util/NetworkTableTestNode.java
rename to networktables/java/src/test/java/test/util/NetworkTableTestNode.java
diff --git a/networktables/java/Athena/src/test/java/test/util/StreamPipeProvider.java b/networktables/java/src/test/java/test/util/StreamPipeProvider.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/test/util/StreamPipeProvider.java
rename to networktables/java/src/test/java/test/util/StreamPipeProvider.java
diff --git a/networktables/java/Athena/src/test/java/test/util/TestExecutor.java b/networktables/java/src/test/java/test/util/TestExecutor.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/test/util/TestExecutor.java
rename to networktables/java/src/test/java/test/util/TestExecutor.java
diff --git a/networktables/java/Athena/src/test/java/test/util/TestServerConnectionProvider.java b/networktables/java/src/test/java/test/util/TestServerConnectionProvider.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/test/util/TestServerConnectionProvider.java
rename to networktables/java/src/test/java/test/util/TestServerConnectionProvider.java
diff --git a/networktables/java/Athena/src/test/java/test/util/TestThreadManager.java b/networktables/java/src/test/java/test/util/TestThreadManager.java
similarity index 100%
rename from networktables/java/Athena/src/test/java/test/util/TestThreadManager.java
rename to networktables/java/src/test/java/test/util/TestThreadManager.java
